src/chromo/models/pythia8.py

```diff
@@ -118,10 +121,20 @@ def _cross_section(self, kin=None):
             st.sigmaAXB,
         )
 
-    def _set_kinematics(self, kin):
-        pythia = self._pythia
-        pythia.settings.resetAll()
+    def _set_decay_mode(self):
+        config = self._config[:]
+        config += [
+            # use our random seed
+            "Random:setSeed = on",
+            # Pythia's RANMAR PRNG accepts only seeds smaller than 900_000_000,
+            # this may change in the future if they switch to a different PRNG
+            f"Random:seed = {self.seed % 900_000_000}",
+            # reduce verbosity
+            "Print:quiet = on",
+        ]
```

Comment:
Please share this common code between set_decay_mode and set_kinematics to keep it DRY. You can add these common options to the config in the init.

Reply:
Moved the code to constructor